src/app/page/Home/Home.test.js

```diff
@@ -46,7 +47,7 @@ describe("Home page", () => {
     expect(page.getByRole("navigation")).toBeInTheDocument();
     expect(page.getByRole("main")).toBeInTheDocument();
     expect(page.getByTestId("btn-login")).toBeInTheDocument();
-    expect(page.getByTestId("btn-signup")).toBeInTheDocument();
+    expect(page.getByTestId("btn-donate")).toBeInTheDocument();
```

Comment:
This test may need revision after the rest of the code changes above.

Reply:
Addressed in 5f11f14. Added additional test cases.